ui/logviewer.html

```diff
@@ -60,7 +60,7 @@
           <li ng-if="isReftest()" ng-cloak>
             <a title="Open the Reftest Analyser in a new window"
                target="_blank" rel="noopener"
-               href="https://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl={{rawLogURL}}&only_show_unexpected=1">
+               href="reftestUrl">
```

Comment:
Sorry I didn't spot this before, but should this be `{{reftestUrl}}` ?

Reply:
Oh shoot!  Technically in React, it's just ``{reftestUrl}`` whereas Angular uses the double braces.  But yeah, what I had was wrong...  OK, I'll hit that in my next little PR.  I have another smaller panel to convert.  Dang, thanks for catching it!  :)

Reply:
Actually, sorry.  You're right.  It's the angular double braces we need here...   I need more coffee...  :)